html/admin/html/form.php

```diff
@@ -10,14 +10,35 @@
 
 <?php if ( ! empty( $key ) ) : ?>
```

Comment:
Force credentials re-validation when `$key` value is still old API key.

Reply:
Fixed in https://github.com/sendsmaily/sendsmaily-wordpress-plugin/pull/11/commits/2f12e97d5d9455b790d44ac16c15bfd90ed332db

Also checking if data allready saved in database when validating credentials

Reply:
Regular expression does not match all API keys. API key can also contain an underscore (`_`). You can use a simpler regular expression, for example `/[^\:]+/` - this every value, except the one which contains colon.

Reply:
Changed to `/^[^\:]+$/`

https://github.com/sendsmaily/sendsmaily-wordpress-plugin/pull/11/commits/a1dc0fde2e32a9ed45e4ecc31cb8c958a9c4e1bf